packages/fx-core/src/plugins/solution/fx-solution/debug/scaffolding.ts

```diff
@@ -135,15 +180,15 @@ export async function scaffoldLocalDebugSettings(
         );
 
         // generate localSettings.json
-        await scaffoldLocalSettingsJson(ctx, inputs, localSettings);
+        await scaffoldLocalSettingsJson(projectSetting, inputs, cryptoProvider, localSettings);
 
         // add 'npm install' scripts into root package.json
         const packageJsonPath = inputs.projectPath;
         let packageJson: any = undefined;
         try {
           packageJson = await PackageJson.load(packageJsonPath);
         } catch (error) {
-          ctx.logProvider?.error(`Cannot load package.json from ${inputs.projectPath}. ${error}`);
+          logProvider?.error(`Cannot load package.json from ${inputs.projectPath}. ${error}`);
```

Comment:
Will `logProvider` be undefined?

Reply:
Updated, validate the v1 context before scaffolding.